### ISSUE TITLE
# in the title

### ISSUE THREAD

**Issue by boldt** [jbenner-radham/rfc-to-bib#4]
If there is an the sign `#` in the title, the printed bibtex entry is not valid. An example:

```
rfc-to-bib 2898
```

Produces 

```
@techreport{rfc2898,
    author       = {B. Kaliski},
    title        = {PKCS #5: Password-Based Cryptography Specification Version 2.0},
    howpublished = {Internet Requests for Comments},
    type         = {RFC},
    number       = 2898,
    pages        = {1-34},
    year         = 2000,
    month        = September,
    issn         = {2070-1721},
    publisher    = {RFC Editor},
    institution  = {RFC Editor},
    url          = {http://www.rfc-editor.org/rfc/rfc2898.txt}
}
```

The `#`-sign in the title should be escaped:

```
    title        = {PKCS \#5: Password-Based Cryptography Specification Version 2.0},
```

**Comment by jbenner-radham** [comment]
Thanks for the heads up. I'll get this addressed in the next release.

**Comment by jbenner-radham** [comment]
The fix has been published in [v0.2.2](https://github.com/jbenner-radham/rfc-to-bib/tree/0.2.2).
